jsonschema2pojo-maven-plugin/example/pom.xml

```diff
@@ -17,7 +17,7 @@
             <plugin>
                 <groupId>org.jsonschema2pojo</groupId>
                 <artifactId>jsonschema2pojo-maven-plugin</artifactId>
-                <version>${project.version}</version>
+                <version>1.2.1</version>
```

Comment:
By fixing the version here, the examples no longer test the current snapshot during integration. Was that the intended behavior?
At least, use a property that could be overridden from the command line.